1.10/tutorials/dcos-101/service-discovery.md

```diff
@@ -11,18 +11,32 @@ menu_order: 4
 <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> Mesosphere does not support this tutorial, associated scripts, or commands, which are provided without warranty of any kind. The purpose of this tutorial is to demonstrate capabilities, and may not be suited for use in a production environment. Before using a similar solution in your environment, you must adapt, validate, and test.</td> </tr> </table>
 
 # Objective
-Your application in the previous part used `redis.marathon.l4lb.thisdcos.directory:6379` as the address for connecting to redis. As redis might be running on any agent in the cluster (and furthermore on different ports), how does this address link to the actual running redis instance?
-In this section, you will learn about DC/OS service discovery by exploring the different options for service discovery for apps in DC/OS.
+Your [app](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.py) in the previous part of this tutorial used `redis.marathon.l4lb.thisdcos.directory` as the address for connecting to Redis, with a port of 6379. As Redis might be running on any agent in the cluster, and potentially on different ports, how does this address resolve to the actual running Redis instance?
 
-# Steps
-  [Service discovery](/docs/1.10/networking/) allows you to connect to your applications without necessarily knowing where they are running. Service discovery is particularly useful in cases where applications may fail and be restarted on a different host.
+In this section, you will learn about DC/OS service discovery by exploring the different options available for apps in DC/OS.
```

Comment:
remove `for apps`. 

Why did you remove the sentence explaining service discovery?
Maybe we can reword to something like

```Service Discovery enables addressing of applications independent of where they are running in the cluster, which is particularly useful in cases where applications may fail and be restarted on a different host.```

Reply:
I see below, maybe the working is still useful...